app/models/solidus_paypal_braintree/gateway.rb

```diff
@@ -266,7 +266,7 @@ def transaction_options(source, options, submit_for_settlement = false)
         params[:shipping] = braintree_shipping_address(options)
       end
 
-      if source.credit_card?
+      if source.credit_card? && options[:bliing_address].present?
```

Comment:
It would be great to have a test for this.

Then you actually would have found your typo as well ;)